hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/migration/StateChildIndices.java

```diff
@@ -27,7 +27,7 @@ public final class StateChildIndices {
     public static final int SPECIAL_FILES = 7;
     public static final int SCHEDULE_TXS = 8;
     public static final int RECORD_STREAM_RUNNING_HASH = 9;
-    public static final int ADDRESS_BOOK = 10;
+    public static final int LEGACY_ADDRESS_BOOK = 10;
```

Comment:
We can add a comment to say `this address book is provided by platform now. And we are just using this as placeholder to not shuffle children.`